internal/handlers/irc/handlers_test.go

```diff
@@ -0,0 +1,209 @@
+package irc
+
+import (
+	"testing"
+
```

Comment:
Is there a reason why there is a space between each import?

Reply:
No there is not, lol. I meant to fix that, will do that soon.